app/src/main/java/com/sun/premierleague/ui/event/EventFragment.kt

```diff
@@ -0,0 +1,59 @@
+package com.sun.premierleague.ui.event
+
+import androidx.core.os.bundleOf
+import com.sun.premierleague.R
+import com.sun.premierleague.base.BaseFragment
+import com.sun.premierleague.data.model.DetailEvent
+import com.sun.premierleague.data.model.Event
+import com.sun.premierleague.utils.RepositoryFactory
+import kotlinx.android.synthetic.main.fragment_event.*
+
+class EventFragment : BaseFragment(), EventContact.View {
+
+    private var presenter: EventContact.Presenter? = null
+    private val adapter = EventAdapter()
+    private var event: Event? = null
```

Comment:
ông dùng lazy được không nhỉ? 
```suggestion
    private var event: Event? by lazy {
        arguments?.getParcelable(BUNDLE_EVENT)
    }
```